Makefile

```diff
@@ -123,3 +123,13 @@ clone-samples:
 		tar -xJvf samples/linux/clean/$$file -C samples/linux/clean; \
 	done
 	tar -xJvf samples/macOS/clean/bincapz.xz -C samples/macOS/clean
+
+ARCH ?= $(shell uname -m)
+CRANE_VERSION=v0.20.2
+install-crane-$(ARCH)-$(CRANE_VERSION):
```

Comment:
To make caching work properly, the Makefile rule should be the output filename that that rule.

For example, `./out/crane-<whatever>`

Reply:
Fixed in [`f5098de` (#438)](https://github.com/chainguard-dev/bincapz/pull/438/commits/f5098deb8d74b545f6cd590d1dd072367054c1e9).